client/src/components/Hero2.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Is this a replacement or in addition to the previous hero class? If it's a replacement we should rename the file Hero, if it's an addition we should rename this and the other Hero into something more descriptive, like CalendarHero

Reply:
It's in addition.  I added a few more hero elements and named them all accordingly.